docs/authentication/saml/overview.mdx

```diff
@@ -1,185 +1,81 @@
 ---
-title: SAML (Beta)
+title: SAML
 description: Clerk supports Enterprise SSO via the SAML protocol so that you can create authentication strategies for Identity Providers, such as Okta.
 ---
 
-# SAML (Beta)
+# SAML
 
-Clerk supports Enterprise SSO via the SAML protocol so that you can create authentication strategies for Identity Providers, such as Okta.
+Clerk supports Enterprise SSO via the SAML protocol, enabling you to create authentication strategies for an Identity Provider (IdP). With Enterprise SSO, users can sign in seamlessly using their IdPs credentials and have their user data synchronized with Clerk.
 
-## Prerequisites
+Currently, Clerk supports the following IdPs for Enterprise SSO:
 
-- A Clerk application with the [Enhanced authentication add-on](https://clerk.com/pricing) plan.
-{/* TODO: This would be a good place for a Nutshell (Ask Rachel). "To do so..." could go in the nutshell. */}
-- An `email_address` enabled as a strategy. In the Clerk Dashboard, go to **User & Authentication > [Email, Phone, Username](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username)**. In the **Contact information** section, enable the **Email address** strategy by toggling on the switch.
+- [Microsoft Azure AD](https://learn.microsoft.com/en-us/entra/fundamentals/whatis)
```

Comment:
I think this list is redundant given we list what we support in the nav for this section.

Also is there a reason we wouldn't link to our own guides instead of external docs, out of curiosity?

Reply:
oh I copied and pasted this from our old overview - you're totally right LOL